app/controllers/amend/DeclarationController.scala

```diff
@@ -16,50 +16,90 @@
 
```

Comment:
The formatting for this file (and many other files in other branches, as I've noticed) has been made worse rather than improving. Can you please disable the scalafmt that is making these changes and recommit please? I have been correcting all the files that I come across but it's not much use if it keeps happening for new files again.

Reply:
Sorry Madhura, intellij formatting has been doing stupid things ever since I started using scalafmt. Will sort out when I get back on Tuesday.

Reply:
I have manually fixed the code formatting for this file. Even when switching to intellij formatting the auto formatter doesn't work. I think in future I will just avoid the autoformat feature entirely - it seems not to work properly at all.